### PR TITLE
[ID] Update `osu!talk`

### PR DESCRIPTION
**Pull request by ZeroPyrozen**
Update osu!talk episodes, discussions, and specials for Indonesian translation.

### CODE REVIEW

File: wiki/osu!talk/id.md [hunk 1]
```diff
@@ -1,112 +1,109 @@
...
-| #5 | Upcoming events Q&A | [Kyonko Hizara](https://osu.ppy.sh/users/444868)and [ztrot](https://osu.ppy.sh/users/6347) | [Tautan](https://www.youtube.com/watch?v=-f9hGuNC4Jg) |
+| 1 | Silakan Tanya Saya Apapun | *N/A* | [Tautan](https://www.youtube.com/watch?v=A0xqa-vBPgY) |
+| 2 | Silakan Tanya Saya Apapun 2015 | [Charles445](https://osu.ppy.sh/users/85000), [Flanster](https://osu.ppy.sh/users/447818), dan [p3n](https://osu.ppy.sh/users/123703) | [Tautan](https://www.youtube.com/watch?v=a9An8cD18Co) |
+| 3 | Perayaan 1 tahun | [Charles445](https://osu.ppy.sh/users/85000) | [Tautan](https://www.youtube.com/watch?v=AHU7RgGCB50) |
+| 4 | Selamat datang kembali | [Kyonko Hizara](https://osu.ppy.sh/users/444868), [Nyquill](https://osu.ppy.sh/users/682935), dan [Zak](https://osu.ppy.sh/users/1375955) | [Tautan](https://www.youtube.com/watch?v=mxXNNH5bnU4) |
+| 5 | Tanya jawab acara yang akan datang | [Kyonko Hizara](https://osu.ppy.sh/users/444868) | [Tautan](https://www.youtube.com/watch?v=-f9hGuNC4Jg) |
+
+## Tautan eksternal
+
+- [Kanal resmi YouTube](https://www.youtube.com/user/osuacademy/videos)
```
Comment (mousesalt) [inline wiki/osu!talk/id.md:108]:
```suggestion
- [Kanal YouTube resmi](https://www.youtube.com/user/osuacademy/videos)
```

Comment (mousesalt) [reply wiki/osu!talk/id.md:108]: that was mean like to youtube's channel instead

**Review by mousesalt — CHANGES_REQUESTED** [review]

File: wiki/osu!talk/id.md [hunk 2]
```diff
@@ -1,112 +1,109 @@
...
-
-### Catch the Beat World Cup 2015
-
-| Tipe | Interviewés | Tautan |
-| :-- | :-- | :-- |
-| Ronde ke- 16 | Dilewat | N/A |
-| Quarter Finals | deadbeat, Tasha and ztrot | [Tautan](https://www.youtube.com/watch?v=lagFdqHqVHA) |
-| Semi Finals | deadbeat, Tasha and ztrot | [Tautan](https://www.youtube.com/watch?v=n6X5fZEEpRA) |
-
+| 1 | Pengerjaan Ulang Staf | [Charles445](https://osu.ppy.sh/users/85000), [Ephemeral](https://osu.ppy.sh/users/102335), dan [OnosakiHito](https://osu.ppy.sh/users/290128) | [Tautan](https://www.youtube.com/watch?v=c10Jiq1xZus) |
```
[the remark this answers is not in the document]
Comment (ZeroPyrozen) [reply wiki/osu!talk/id.md:69]: I'm not sure about this one, other translated language translate most of the topic but few of them are leave as it is.

There should be original topic and alternate topic title columns like these example:

1. [Netflix - Do I need to translate episode titles for Korean timed text assets?](https://partnerhelp.netflixstudios.com/hc/en-us/articles/115000783548-Do-I-need-to-translate-episode-titles-for-Korean-timed-text-assets-)
2. [IMDb - Alternate titles (AKAs)](https://help.imdb.com/article/contribution/titles/alternate-titles-akas/GBFBWTQG2RLMHSUR?ref_=helpsect_pro_4_8#)
3. [List of Lucifer episodes](https://en.wikipedia.org/wiki/List_of_Lucifer_episodes) | [Lucifer (Fernsehserie)/Episodenliste](https://de.wikipedia.org/wiki/Lucifer_(Fernsehserie)/Episodenliste)

**Review by mousesalt — DISMISSED** [review]

File: wiki/osu!talk/id.md [hunk 3]
```diff
@@ -1,112 +1,109 @@
...
+| 45 | [Rafis](https://osu.ppy.sh/users/2558286) | Pemain pro osu! | [Tautan](https://www.youtube.com/watch?v=wKhuovIMa8k) |
+| 46 | [Spare](https://osu.ppy.sh/users/2204373) | Pemain top osu! di Britania Raya | [Tautan](https://www.youtube.com/watch?v=MTWgwsIxPRc) |
+| 47 | [Helblinde](https://osu.ppy.sh/users/48053) | Featured artist | [Tautan](https://www.youtube.com/watch?v=cviwU4xkM-w) |
+| 48 | [MatsumotoRise](https://osu.ppy.sh/users/672726) | Pemain pro osu! | [Tautan](https://www.youtube.com/watch?v=8-3d2ZHw2O4) |
+| 49 | [Toy](https://osu.ppy.sh/users/2757689) | Pemain pro osu! | [Tautan](https://www.youtube.com/watch?v=lI8mIJLOu_k) |
 
 ## Diskusi
 
-| EP\# | Topik | Orang yang diwawancara | Tautan |
+| EP No. | Topik | Narasumber | Tautan |
```
Comment (AncuL001) [inline wiki/osu!talk/id.md:67]:
```suggestion
| No. EP | Topik | Narasumber | Tautan |
```

File: wiki/osu!talk/id.md [hunk 4]
```diff
@@ -1,112 +1,109 @@
...
+| 24 | Restrukturisasi Beatmap Nominator | [pishifat](https://osu.ppy.sh/users/3178418), [Seijiro](https://osu.ppy.sh/users/2581696), [Sonnyc](https://osu.ppy.sh/users/11771), [Xexxar](https://osu.ppy.sh/users/2773526), dan [Yuii-](https://osu.ppy.sh/users/2935923) | [Tautan](https://www.youtube.com/watch?v=Va0cHczm8Mk) |
+| 25 | Aturan Main yang dibuat oleh Komunitas | [Shawdooow](https://osu.ppy.sh/users/7726082), [ColdVolcano](https://osu.ppy.sh/users/7492333), dan [andy840119](https://osu.ppy.sh/users/1030492) | [Tautan](https://www.youtube.com/watch?v=0nuLwWejdII) |
+| 26 | osu!idol 2017 | [Flanster](https://osu.ppy.sh/users/447818), [Kyonko Hizara](https://osu.ppy.sh/users/444868), [CptHampton](https://osu.ppy.sh/users/3200587), dan [HeatKai](https://osu.ppy.sh/users/332555) | [Tautan](https://www.youtube.com/watch?v=Xz2R18xeqhI) |
 
 ## Spesial
 
-| EP\# | Acara | Orang yang diwawancara | Tautan |
+| EP No. | Topik | Narasumber | Tautan |
```
Comment (AncuL001) [inline wiki/osu!talk/id.md:98]:
```suggestion
| No. EP | Topik | Narasumber | Tautan |
```

**Review by AncuL001 — APPROVED** [review]
the rest seems ok

**Review by mousesalt — APPROVED** [review]
this looks fine 👍 great

**Review by TicClick — APPROVED** [review]

**Pull request merged**